base-notebook/Dockerfile

```diff
@@ -38,7 +38,8 @@ WORKDIR /tmp
 RUN mamba install --yes \
     'notebook' \
     'jupyterhub' \
-    'jupyterlab' && \
+    'jupyterlab' \
+    'nbclassic' && \
```

Comment:
It seems that we need to install this explicitly, and `nbclassic` is kept to make migration easier. So, our images should also support `nbclassic`.
https://jupyter-notebook.readthedocs.io/en/stable/migrating/multiple-interfaces.html#nbclassic-and-notebook-7

Reply:
That's right. The classic notebook UI is then available under the `/nbclassic` endpoint.